clients/admin-ui/src/features/common/Icon/Add.tsx

```diff
@@ -0,0 +1,32 @@
+import { createIcon } from "@fidesui/react";
+
+export default createIcon({
+  displayName: "AddIcon",
+  viewBox: "0 0 40 40",
+  path: (
+    <svg
+        width="40"
+        height="40"
+        viewBox="0 0 40 40"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <g clipPath="url(#clip0_84_5307)">
```

Comment:
Is this just a random string we use here?

Reply:
@earmenda This is the copy/paste as SVG from Figma. If you hover over an icon in Figma and right click, you can copy the SVG data. This came from that data.

Reply:
I was adjusting the system logo icons and noticed that figma's clip path wasn't adding anything to the icon. Might try deleting it to see if it's contributing anything.